TelepresenceBot/core/src/main/java/com/novoda/tpbot/support/Observable.java

```diff
@@ -33,22 +36,26 @@ protected synchronized void notifyOf(T newValue) {
         }
     }
 
-    protected synchronized void detachObservers() {
+    public synchronized void detachObservers() {
         observers.clear();
     }
 
-    protected synchronized void setChanged() {
-        hasChanged = true;
-    }
+    public abstract Observable<T> start();
```

Comment:
Can we move the abstract method as the first class method so it's easier to understand what needs implementing?